lib/getPostcssResult.cjs

```diff
@@ -25,9 +26,10 @@ async function getPostcssResult(stylelint, { customSyntax, filePath, code } = {}
 		return cached;
 	}
 
+	const fix = normalizeFixMode(stylelint._options, stylelint._options.config);
 	const syntax = await (customSyntax
 		? getCustomSyntax(customSyntax, stylelint._options.configBasedir)
-		: cssSyntax(stylelint._options.fix));
+		: cssSyntax(fix));
```

Comment:
@ybiquitous I don't think it will matter much, but this part is not a refactor.
i.e. `config.fix` used to be ignored

So either it was deliberately done and it needs a comment or it was a careless omission.

Reply:
@Mouvedia Good catch. Since this is a behavior change, I think we should separate it from this PR to make it easier to track the commit history. 👍🏼 

At a glance, it seems that we should support the `fix: true` config also here, in addition to `--fix`, but I'm not 100% sure. Can you confirm how Stylelint behaves when `fix: true` config is specified?

Reply:
> Can you confirm how Stylelint behaves when `fix: true` config is specified?

Let's postpone that to the new issue/PR.

> I think we should separate it from this PR…

@ryo-manba you were right in trying to support both `config.fix` and `fix`, nonetheless we are trying to have commits that can be reverted as a whole. [We might also have to support `configFile` to be thorough.](https://github.com/stylelint/stylelint/pull/8106#discussion_r1844933085)
Could you only pass `LinterOptions` as a single argument to `normalizeFixMode `?
It will be fixed separately.

Reply:
Fixed in this commit.
[Config is not supported](https://github.com/stylelint/stylelint/pull/8106/commits/6c071ee93bd21e01f7ce9d9b59160a79f416803b)

Reply:
@ybiquitous see https://github.com/stylelint/stylelint/pull/8106/files/f59eade747442f97d3257ad85954c20be8badbcb..6c071ee93bd21e01f7ce9d9b59160a79f416803b#r1845332494